src/scenarios/replacement-cycling.py

```diff
@@ -0,0 +1,439 @@
+#!/usr/bin/env python3
```

Comment:
file needs to be named with underscore not hyphen. rpc_test.py fails locally for me without that fix, dunno why CI is ok with it 🤷

Reply:
I thought the scenarios test would have caught that.